<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLibSearchTitle</class>
 <widget class="QWidget" name="MLibSearchTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btRefresh">
        <property name="text">
         <string>refresh</string>
        </property>
        <property name="icon">
         <iconset resource="resfile.qrc">
          <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbLimit">
        <property name="text">
         <string>limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MLabel" name="label" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btQuery">
        <property name="text">
         <string>search</string>
        </property>
        <property name="icon">
         <iconset resource="resfile.qrc">
          <normaloff>:/new/icons/icons/work.png</normaloff>:/new/icons/icons/work.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+E</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fmPW">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rbWord">
           <property name="toolTip">
            <string>word</string>
           </property>
           <property name="text">
            <string>wrd</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbPhrase">
           <property name="toolTip">
            <string>phrase</string>
           </property>
           <property name="text">
            <string>ph</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbPhraseSpc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>accept spaces</string>
           </property>
           <property name="text">
            <string>spc</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btHide">
        <property name="toolTip">
         <string>hide</string>
        </property>
        <property name="text">
         <string>hide</string>
        </property>
        <property name="icon">
         <iconset resource="resfile.qrc">
          <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmLimit">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>page</string>
        </property>
        <property name="buddy">
         <cstring>spnPage</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnPage">
        <property name="toolTip">
         <string>limits results as from limit*page to limit*(page+1)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>limit</string>
        </property>
        <property name="buddy">
         <cstring>spnLimit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnLimit">
        <property name="toolTip">
         <string>limits results as from limit*page to limit*(page+1)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>MLabel</class>
   <extends>QWidget</extends>
   <header>mlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btRefresh</tabstop>
  <tabstop>spnPage</tabstop>
  <tabstop>spnLimit</tabstop>
  <tabstop>rbWord</tabstop>
  <tabstop>rbPhrase</tabstop>
  <tabstop>cbPhraseSpc</tabstop>
  <tabstop>btHide</tabstop>
 </tabstops>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
